web/client/components/geostory/media/Map.jsx

```diff
@@ -61,7 +63,18 @@ export default compose(
                 id: `media-${id}`
             }} // if map id is passed as number, the resource id, ol throws an error
             layers={layers}
-            options={applyDefaults(options)}
+            options={
+                // mouseWheelZoom is enabled only if inlineEditing is active and zoomControl too
+                applyDefaults(!isContentFocused ? {mapOptions} : {
+                    mapOptions: {
+                        ...mapOptions,
+                        interactions: {
```

Comment:
You should not activate zoom wheel when editing. make it a normal option like the others:
 it simplifies a lot and give a feature more to the user.

Reply:
@tdipisa what about this?

Reply:
@MV88 @offtherailz no because if you include the mouse wheel in settings then the user expect to have it also in the story where the wheel must be disabled for scrolling purposes.